.travis.yml

```diff
@@ -1,15 +1,37 @@
 language: php
 
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+
 php:
   - 5.3
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
 
-before_script: composer install --dev
+matrix:
+  include:
+    - php: 5.6
+      env: SYMFONY_VERSION="2.3.*"
+    - php: 5.6
+      env: SYMFONY_VERSION="2.7.*"
+    - php: 5.6
+      env: SYMFONY_VERSION="2.8.*"
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.*"
+    - php: 5.6
+      env: SYMFONY_VERSION="3.1.*"
+  allowed_failure:
+    - env: SYMFONY_VERSION="3.1.*"
```

Comment:
this one should be better handled with a job testing against dev versions of dependencies